CONTRIBUTING.md

```diff
@@ -0,0 +1,45 @@
+# Contribution guideline
```

Comment:
@bensternthal does OpenJS have a standard contribution guide we should incorporate into this?

Reply:
Not that I am aware of. I think most projects customize the contribution guidelines to the specific needs of the project.

You can see the full list of contribution guides [here](https://github.com/openjs-foundation/cross-project-council).